change/@ni-nimble-components-30881da7-4a97-48ad-81c0-db79e1c01c98.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The switch and radio group also support labels. Do we expect them to behave the same way? If so, could you add tests for them too? 

If there's tech debt around those labels that makes them behave differently, no need to grow the scope of this PR and fix that now (as long as that tech debt is tracked somewhere).

Reply:
I've added a story for the switch and the radio group.